### PR TITLE
klientctl: improve install script

### PR DESCRIPTION
**Pull request by rjeczalik**

### CODE REVIEW

**Comment by cihangir** [comment]
LGTM

File: go/src/koding/klientctl/install-kd.sh [hunk 1]
```diff
@@ -102,6 +106,40 @@ install_vagrant_deps() {
...
 
+test_vagrant() {
+  echo "VirtualBox and Vagrant versions must match in order" 1>&2
+  echo "to be able to build a Vagrant box." 1>&2
+  echo 1>&2
+  echo "Vagrant 1.7.x expects VirtualBox 4.x versions and" 1>&2
+  echo "Vagrant 1.8.x expects VirtualBox 5.x versions." 1>&2
+  echo 1>&2
+
+  if ! prompt_install "Do you want to test building Vagrant box with VirtualBox provider?"; then
```
Comment (leeola) [inline go/src/koding/klientctl/install-kd.sh]: Ran into an issue here i think, my script didn't continue installing vagrant - should it?

_(I'm talking with Rafal now about it, just didn't want an accidental merge if there's an issue, so documenting it here)_

Comment (rjeczalik) [inline go/src/koding/klientctl/install-kd.sh]: You're correct sir, it does exit for me to, eventough the last command exited with 0:

```
_____________________________________________________________________

Hello
+ vagrant destroy -f
==> default: Forcing shutdown of VM...
==> default: Destroying VM and associated drives...
+ popd
/Volumes/rjk.io/src/github.com/koding/koding
+ echo

+ echo

+ echo 'Building Vagrant box was successful'
Building Vagrant box was successful
+ echo

```

Fixing.

Comment (rjeczalik) [inline go/src/koding/klientctl/install-kd.sh]: @leeola ptal

**Comment by leeola** [comment]
LGTM

File: go/src/koding/klientctl/install-kd.sh (hunk 1 above)
Comment (leeola) [inline go/src/koding/klientctl/install-kd.sh]: :+1:

**Pull request merged**